editor.planx.uk/src/pages/FlowEditor/lib/store/navigation.ts

```diff
@@ -115,19 +118,28 @@ export const navigationStore: StateCreator<
   },
 
   /**
-   * Calculate the status of each section node based on the current position in a flow
-   *   ** will need to become aware of all breadcrumbs within a section in the near future for "back" rather than only currentCard & upcomingCardIds
+   * Calculate the status of each section node based on the current position in a flow (eg state.cachedBreadcrumbs) or provided breadcrumbs (eg on reconciliation)
+   *   Pass updatedSectionNodeIds on reconiliation to ensure we set an accurrate status for every previously seen section because updated sections are removed from breadcrumbs during reconciliation
    */
-  sectionStatuses: (): Record<string, SectionStatus> => {
+  sectionStatuses: (
+    breadcrumbs?: Store.breadcrumbs,
+    updatedSectionNodeIds?: string[]
+  ): Record<string, SectionStatus> => {
     const { sectionNodes, currentCard, upcomingCardIds, cachedBreadcrumbs } =
       get();
 
+    // Default to cachedBreadcrumbs unless a breadcrumbs-like object is explicitly provided (eg on reconciliation, before app state has been updated)
+    if (!breadcrumbs) breadcrumbs = cachedBreadcrumbs;
+
     const sectionStatuses: Record<string, SectionStatus> = {};
     Object.keys(sectionNodes).forEach((sectionId) => {
-      if (
+      if (updatedSectionNodeIds?.includes(sectionId)) {
+        // We only expect to receive updatedSectionNodeIds argument on reconciliation, therefore
+        //   this status should never apply to regular forwards/back/change navigation
+        sectionStatuses[sectionId] = SectionStatus.NeedsUpdated;
+      } else if (
```

Comment:
Ben's work will keep building on this by showing this status for all sections containing updated questions - but a basic way to _already_ trigger this status is to: save, directly edit your section node in your flow (eg give it a new title), re-publish, resume! 
![Screenshot from 2023-03-16 08-46-08](https://user-images.githubusercontent.com/5132349/225550191-ba9ef6e5-8664-444b-a6e7-ab09c64566fa.png)